cfme/physical/physical_server.py

```diff
@@ -68,58 +69,90 @@ def load_details(self, refresh=False):
             view.browser.refresh()
             view.flush_widget_cache()
 
-    def execute_button(self, button_group, button, handle_alert=False):
+    def execute_button(self, button, option, handle_alert=False):
         view = navigate_to(self, "Details")
-        view.toolbar.custom_button(button_group).item_select(button, handle_alert=handle_alert)
+        view.toolbar.custom_button(button).item_select(option, handle_alert=handle_alert)
+        return view
 
-    def power_on(self):
-        view = navigate_to(self, "Details")
-        view.toolbar.power.item_select("Power On", handle_alert=True)
+    def __execute_action_button(self, button, option, handle_alert=True, **kwargs):
```

Comment:
**REQUIRED**
Only one underscore is required to designate a method as private.
`_private_method()`

Reply:
OK :confused: